src/sentry/integrations/vercel/integration.py

```diff
@@ -297,12 +287,16 @@ def update_env_variable(self, client, vercel_project_id, data):
             f"Could not update environment variable {key} in Vercel project {vercel_project_id}."
         )
 
+    def uninstall(self):
+        client = self.get_client()
+        client.uninstall(self.get_configuration_id())
```

Comment:
@MeredithAnya shouldn't we update the `Integration` to remove the configuration ID? I suspect this might cause bugs if someone uninstalls the integration on one that's installed on multiple orgs.

Reply:
We still get the webhook if we uninstall via the API, so it should still ultimately update the integration